src/admin/mod.rs

```diff
@@ -380,6 +382,126 @@ async fn handle_metrics(reg: Arc<Mutex<Registry>>, _req: Request<Body>) -> Respo
         .unwrap()
 }
 
+// we mirror envoy's behavior: https://www.envoyproxy.io/docs/envoy/latest/operations/admin#post--logging
+//curl -X POST http://127.0.0.1:15021/logging To list the loggers
+//curl -X POST http://127.0.0.1:15021/logging?level={new_level} To change the logging level across all loggers
+
+//TODO: below is not implemented
+//curl -X POST http://127.0.0.1:15021/logging?{mod_name}={new_level} To change a particular logger’s level
+//curl -X POST http://127.0.0.1:15021/logging?paths={mod_name_1}:{level1},{mod_name_2}:{level2} To change multiple logging levels at once
+
+//NOTE: mutilple query parameters is not supported, for example
+//curl -X POST http://127.0.0.1:15021/logging?"tap=debug&router=debug"
+static HELP_STRING: &str = "
+usage: loglevel: error|warn|info|debug|trace|off
+usage: POST /logging (list current level)
+usage: POST /logging?level=<level> (change all levels)
+";
+async fn handle_logging(req: Request<Body>) -> Response<Body> {
+    match *req.method() {
+        hyper::Method::POST => {
+            let params: HashMap<String, String> = req
+                .uri()
+                .query()
+                .map(|v| {
+                    url::form_urlencoded::parse(v.as_bytes())
+                        .into_owned()
+                        .collect()
+                })
+                .unwrap_or_else(HashMap::new);
+
+            match params.len() {
+                // POST /logging, list all the loggers
+                0 => list_loggers(),
+                // POST /logging/?{query_param}, change the loggers
+                1 => {
+                    //POST /logging?level={new_level}
+                    if let Some(global_level) = params.get_key_value("level") {
+                        change_global_logger(global_level.1.into())
+                    }
+                    //POST /logging?paths={mod_name_1}:{level1},{mod_name_2}:{level2}
```

Comment:
might be nice to have this be `?level` still?

Reply:
hi John, @howardjohn 
I implemented the functionality to set global and specific mod's log level, meanwhile, I found the eventfilter.parse() and directive.parse() did a perfect job for us to parse the query parameters, so the code is simpler than before. 
Now the queries looks like this :
```
usage: POST /logging                                            (To list current level)
usage: POST /logging?level=<level>                              (To change global levels)
usage: POST /logging?level={mod1}:{level1},{mod2}:{level2}      (To change specific mods' logging level)
```
I hope this is what you want.

meanwhile, I merge the code with your PR #265, in the telemetry::fmt_layer(), my modification is to get the reload_handle for future usage, it does not quite conflict with yours, I re-tested my cases and it went well.

I put all the test case at the description of this PR, please review. Thank you.